app/models/direct_file_data.rb

```diff
@@ -612,6 +616,39 @@ def build_new_qualifying_child_information_node
     parsed_xml.css('QualifyingChildInformation').last.add_next_sibling(dd.to_s)
   end
 
+  def third_party_designee
+    df_xml_value(__method__)
+  end
+
+  def third_party_designee=(value)
+    create_or_destroy_df_xml_node(__method__, true, 'ThirdPartyDesigneeInd')
```

Comment:
I think technically if `value` is `nil` then you want to pass `false` instead of `true` here, but it seems unlikely that this setter will ever be used to delete the node, so it's not important to fix this.

Reply:
all other setters are set to true. so keeping it consistent here. this is only used in test scenarios

Reply:
yeah, that's fine. I don't think our tests ever delete direct file nodes, so keeping it always true doesn't ever cause a problem. But if we did want to delete these fields/nodes, we would have to update this code to be conditional instead of hard coded. It's definitely not a problem right now though.